test/integration/keyboard.integration.ts

```diff
@@ -17,6 +17,17 @@ test.describe('Tests served by: fixture-file rendering by Nebula', () => {
   let expectation: Collection;
 
   const environment = new NebulaFixture(senseHorizon('light'), 'light', 'en-EN');
+  const focusedText = () => page.locator('*:focus').innerText();
+  const isSVG = async () => {
+    const index = (await page.locator('*:focus').innerHTML()).match(/svg/)?.index;
+    if (index && index > 0) return true;
+    return false;
+  };
+  const isOption = async () => {
+    const index = (await page.locator('*:focus').innerHTML()).match(/option/)?.index;
+    if (index && index > 0) return true;
+    return false;
+  };
```

Comment:
There aren't really any util functions defined in this file. Do they fit into some other file or maybe create a new one?

Reply:
Okay, I will create a new file to relocate them.